src/toucan2/jdbc/result_set.clj

```diff
@@ -142,6 +145,7 @@
         col-names         (get builder :cols (next.jdbc.rs/get-modified-column-names
                                               (.getMetaData rset)
                                               combined-opts))
+        col-names-kw      (vec (keep #(column-name->keyword % label-fn) col-names))
```

Comment:
If we're really worried about performance, `(into [] (keep ...) ...)` would be faster, right?

Reply:
This code only runs once per ResultSet, so it's not as critical to make optimizations here. But sure, there's no harm in switching to a transducer.